two_factor_otp_auth/README.rst

```diff
@@ -0,0 +1,19 @@
+================================
+Ventor Two Factor Authentication
+================================
+
+
+
+
+Credits
```

Comment:
@Werded , please, add readme file similar to the https://github.com/OCA/mis-builder/blob/10.0/mis_builder/README.rst 
Note that some sections are not needed, of course.

Reply:
done